### DISCUSSION TITLE
Pressur regulator

### DISCUSSION THREAD

**Discussion by Yahiaassam** [OpenModelica/OpenModelica#9003] [Q&A] [answered]
Hello everyone, I have a question about how to model a pressure regulator in open modelica?

**Comment by AnHeuermann** [comment, marked as answer]
Hey @Yahiaassam.
It appears your question is very broad and not specific to one Modelica tool, so you might get better answers in a forum for general Modelica questions, e.g. [StackOverflow](https://stackoverflow.com/questions/tagged/modelica).

**Reply by Yahiaassam** [reply]
hey @AnHeuermann ,
thanks for your reply, my question it is about to reduse an closedtanke pression for exemple from 350 bars to 8 bars
